### ISSUE TITLE
BDD API reference is not shown

### ISSUE THREAD

**Issue by weirdgiraffe** [chaijs/chaijs.github.io#159]
http://www.chaijs.com/api/bdd/ page is almost empty and there is completely no references on it

**Comment by tzkmx** [comment]
It looks like the linked script for chai returns 404: `https://cdn.rawgit.com/chaijs/chai//chai.js` <- look the double slash before the file name.

**Comment by duckbrain** [comment]
Maybe a path like `https://raw.githubusercontent.com/chaijs/chai/master/chai.js` would be better. [RawGit](https://rawgit.com/) says the other format is for development and is rate limited.

**Comment by wyckster** [comment]
The double slash shows that the hash/key/branch fragment is missing from the API url.

It should look something like this:
https://cdn.rawgit.com/chaijs/chai/master/chai.js

@duckbrain, I disagree.  I looked at that RawGit link, and it says that the cdn.rawgit.com api is the one to use in production:

> Use this URL in production
> 
> https://cdn.rawgit.com/chaijs/chai/master/chai.js
> 
> No traffic limits or throttling. Files are served via StackPath's super fast global CDN.
> 
> Use a specific tag or commit hash in the URL (not a branch). Files are cached permanently based on the URL. Query strings are ignored.
> 
> The catch: this is a free service, so there are no uptime or support guarantees.
>

**Comment by vsr** [comment]
The script url in the [template](https://github.com/chaijs/chaijs.github.io/blob/89f037733a41e1628d865718b2eb2ca9f9689ed3/_includes/head.html) is something like 
`<script src="https://cdn.rawgit.com/chaijs/chai/{{ site.data.releases[0].tag_name }}/chai.js"></script>`
So, assuming that something went wrong during deployment and the site data wasn't loaded.

Could you please check @keithamus, @logicalparadox ?

**Comment by lucasfcosta** [comment]
Hey everyone,

Our bot was pushing the built `_data` to `gh-pages` while the website is being built from `master` due to this being the "home" repo for the `chaijs` org.

Should be g2g once #163 is merged.

That also fixes our build proccess by avoiding to push to `master` when the build has not been triggered directly by a merge to that branch.

The `bot` is also now the only user able to push directly to `master`.

**Comment by akikoskinen** [comment]
Not wanting to make another issue for this, but at least these pages are broken too:

- http://www.chaijs.com/api/assert/
- http://www.chaijs.com/plugins/

**Comment by lucasfcosta** [comment]
@akikoskinen This is caused by the same reason I mentioned above. There are no `_data` files on `master`, which is from where the site is being built from.

**Comment by lucasfcosta** [comment]
**Fixed.**

![How I feel after solving this](https://media.giphy.com/media/2HFBvKS9zmkEw/giphy.gif)
